src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -18,27 +18,31 @@ export default function ColorPickerControl({
     ...rest
 }: Props): JSX.Element | null {
     const paletteRef = React.useRef<HTMLDivElement>(null);
+    const toggleRef = React.useRef<HTMLButtonElement>(null);
     const [isColorPickerToggled, setIsColorPickerToggled] = React.useState(false);
-    const [isPaletteActive, handlers] = useAttention();
+    const [handlers] = useAttention();
+    const { current: paletteEl } = paletteRef;
+    const { current: toggleEl } = toggleRef;
 
     const handleSelect = (color: string): void => {
         setIsColorPickerToggled(false);
         onColorSelect(color);
     };
 
     const handleBlur = ({ relatedTarget }: React.FocusEvent<HTMLButtonElement>): void => {
-        const { current: paletteEl } = paletteRef;
         // IE11 does not have relatedTarget but update activeElement before blur
         const nextTarget = relatedTarget || document.activeElement;
 
-        if (isPaletteActive || (nextTarget && paletteEl && paletteEl.contains(nextTarget as Node))) {
+        if ((nextTarget && paletteEl && paletteEl.contains(nextTarget as Node)) || toggleEl === (nextTarget as Node)) {
             return;
         }
 
         setIsColorPickerToggled(false);
     };
 
-    const handleClick = (): void => setIsColorPickerToggled(!isColorPickerToggled);
+    const handleClick = (): void => {
```

Comment:
Is this change needed or can we revert it?

Reply:
forgot that, thanks